new-lamassu-admin/src/pages/Maintenance/CashCassettes.js

```diff
@@ -166,10 +166,10 @@ const CashCassettes = () => {
         action: 'setCassetteBills',
         deviceId: id,
         cashbox,
-        cassette1,
-        cassette2,
-        cassette3,
-        cassette4
+        cassette1: cassette1 ?? 0,
+        cassette2: cassette2 ?? 0,
+        cassette3: cassette3 ?? 0,
+        cassette4: cassette4 ?? 0
```

Comment:
You could move this to the `onContinue()` function of the Wizard, after the `parseInt()`.